group-coordinator/src/main/java/org/apache/kafka/coordinator/group/consumer/ConsumerGroup.java

```diff
@@ -434,22 +437,36 @@ public void setSubscriptionMetadata(
     public Map<String, TopicMetadata> computeSubscriptionMetadata(
         ConsumerGroupMember oldMember,
         ConsumerGroupMember newMember,
-        TopicsImage topicsImage
+        TopicsImage topicsImage,
+        ClusterImage clusterImage
```

Comment:
Would it make sense to directly pass `MetadataImage`?

Reply:
Since metadataImage has a lot more images that we don't need and passing the specific arguments makes it more readable, maintainable, and testable I feel like we can keep it as two separate arguments.